app/components/print-view-controls.js

```diff
@@ -29,7 +27,7 @@ export default class PrintViewControls extends Component {
       event_action: 'Disabled print view',
     });
     // GA
-    this.get('metrics').trackEvent('MatomoTagManager', {
+    this.metrics.trackEvent('MatomoTagManager', {
```

Comment:
This should be fine? I'm assuming changing away from `.get()` is just an Ember modernization thing so David's changes should still be fine here.